Browncoats.rte/Devices/Explosives/FuelBomb/FuelBomb.lua

```diff
@@ -2,63 +2,111 @@ function Create(self)
 
 	self.explodeTimer = Timer();
 	self.partList = {};
-	self.burnList = {};
 
 	self.explodeTime = 2000;
-	self.numOfParticles = 50;
-	self.particlesPerOil = 3;
+	self.numOfParticles = 40;
+	self.particlesPerOil = 5;
 
 	self:EraseFromTerrain();
-	self.Scale = 0;
 
 	for i = 1, self.numOfParticles do
 		self.partList[i] = CreateMOPixel("Browncoat Fuel Bomb Fuel");
 		self.partList[i].Pos = self.Pos;
-		self.partList[i].Vel = Vector((math.random()*20),0):RadRotate(6.28318*math.random()) + self.Vel;
+		self.partList[i].Vel = Vector(math.random(20), 0):RadRotate(6.283 * math.random()) + self.Vel;
```

Comment:
Is there a reason for the precision change on 2pi?

Reply:
Good catch, we should really use math.pi * 2 here anyway.